CPT331.WebAPI/App_Start/SwaggerConfig.cs

```diff
@@ -0,0 +1,246 @@
+using System.Web.Http;
+using WebActivatorEx;
+using CPT331.WebAPI;
+using Swashbuckle.Application;
+
+[assembly: PreApplicationStartMethod(typeof(SwaggerConfig), "Register")]
+
```

Comment:
The advantage is you don't need to understand the API to use swagger. In postman you need to know the routes in order to perform an operation. Swagger looks for all classes that inherit ApiController and provides an interface to access and test it. 

It doesn't add a lot of code, other than Startup method in Global.asax,cs and the SwaggerConfig class in App_Start.

In short, it is just swagger easier to use than Postman.